internal/oci/uvm.go

```diff
@@ -327,6 +238,26 @@ func handleSecurityPolicy(ctx context.Context, a map[string]string, lopts *uvm.O
 	}
 }
 
+// sets options common to both WCOW and LCOW from annotations
+func specToUVMCreateOptionsCommon(ctx context.Context, opts *uvm.Options, s *specs.Spec) error {
```

Comment:
Worth having it return an error if it can't fail?

Reply:
It was pre-emptive on my part; I didnt know if future parsing would error somewhere.

Reply:
Gotcha, resolving!

Reply:
Oh, didn't realize this wasn't changed in the latest round. I'd say let's remove the error for now unless its needeed